ttg/ttg/serialization/std/list.h

```diff
@@ -0,0 +1,43 @@
+//
+// Created by Eduard Valeyev on 5/11/21.
+//
+
+#ifndef TTG_SERIALIZATION_STD_LIST_H
+#define TTG_SERIALIZATION_STD_LIST_H
+
+#include "ttg/serialization/std/allocator.h"
+#include "ttg/serialization/traits.h"
+
+#ifdef TTG_SERIALIZATION_SUPPORTS_MADNESS
+// MADNESS supports std::list serialization by default
+#endif
+
+#ifdef TTG_SERIALIZATION_SUPPORTS_BOOST
+#include <boost/serialization/list.hpp>
+
+namespace ttg::detail {
+  template <typename Archive, typename T, typename A>
+  inline static constexpr bool is_stlcontainer_boost_serializable_v<Archive, std::list<T, A>> =
+      is_boost_serializable_v<Archive, T>&& is_boost_serializable_v<Archive, A>;
```

Comment:
Nitpick: the formatting here is weird:

```suggestion
      is_boost_serializable_v<Archive, T> && is_boost_serializable_v<Archive, A>;
```

Reply:
indeed ... let me check why clang-format does that (or if it somehow was not invoked by git-hooks)

Reply:
missing space is indeed what the clang-format 11.1.0 on my Mac produces with our `.clang-format` ...